src/azure-cli/azure/cli/command_modules/feedback/custom.py

```diff
@@ -440,34 +441,20 @@ def _get_az_version_summary():
     """
     az_vers_string = get_az_version_string()[0]
 
-    lines = az_vers_string.splitlines()
+    # Remove consecutive spaces
+    import re
+    az_vers_string = re.sub(' +', ' ', az_vers_string)
 
+    # Add each line until 'python location'
+    lines = az_vers_string.splitlines()
     new_lines = []
-    ext_line = -1
-    legal_line = -1
     for i, line in enumerate(lines):
-        if line.startswith("azure-cli"):
-            line = " ".join(line.split())
-            new_lines.append(line)
-        if line.lower().startswith("extensions:"):
-            ext_line = i
-            continue
-        l_lower = line.lower()
-        if all(["legal" in l_lower, "docs" in l_lower, "info" in l_lower]):
-            legal_line = i
+        if 'python location' in line.lower():
             break
+        new_lines.append(line)
 
-    new_lines.append("")
-
-    if 0 < ext_line < legal_line:
-        for i in range(ext_line, legal_line):
-            l_lower = lines[i].lower()
-            if "python location" in l_lower or "extensions directory" in l_lower:
-                break
-
-            line = " ".join(lines[i].split())
-            new_lines.append(line)
-
```

Comment:
There is no need to parse the output of `az --version` in such detail. Simply remove consecutive spaces and it will work, given we don't need to minify the body of `az feedback` anymore (https://github.com/Azure/azure-cli/pull/17353).